lib/Doctrine/DBAL/Migrations/Tools/Console/Command/DiffCommand.php

```diff
@@ -90,10 +90,10 @@ public function execute(InputInterface $input, OutputInterface $output)
         $toSchema = $this->getSchemaProvider()->createSchema();
 
         //Not using value from options, because filters can be set from config.yml
-        if ( ! $isDbalOld && $filterExpr = $conn->getConfiguration()->getFilterSchemaAssetsExpression()) {
+        if (!$isDbalOld && $filterExpr = $conn->getConfiguration()->getFilterSchemaAssetsExpression()) {
```

Comment:
this change does not match the Doctrine coding standards.

I would be totally in favor of using PSR-2 compatible CS rules, but this should be an educated decision.